### PR TITLE
Add multiclass classification dataset & set up basic model-assessment tests

### PR DESCRIPTION
**Pull request by romanlutz**
## Description

To properly check various cases (including edge cases) we need these tests for model-assessment the same way they already exist for fairness and interpret. This PR adds a very basic set of tests for "Model overview" for multiclass classification (with the newly added wine dataset), binary classification (adult census income), and regression (boston).

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [ ] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [ ] raiwidgets
- [ ] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [ ] No new tests required.
- [x] New tests for the added feature are part of this PR.
- [x] I validated the changes manually.

## Screenshots (if appropriate):

New dataset for multiclass classification:

image

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/dashboard/src/error-analysis/__mock_data__/dummyTreeWine.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IErrorAnalysisTreeNode } from "@responsible-ai/core-ui";
```
Comment (imatiach-msft) [inline apps/dashboard/src/error-analysis/__mock_data__/dummyTreeWine.ts:4]: Some of these test files are quite large (like the one below with 4k lines). I wonder if we can move them to a zip folder on blob storage at some point. Such large autogenerated test data doesn't really belong in git I think.  Also it makes prettier extensions really slow.

File: apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeAxisConfigDialog.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function describeAxisConfigDialog(): void {
```
[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeAxisConfigDialog.ts:4]: +1 on this

File: apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeModelPerformanceStats.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IModelAssessmentData } from "../IModelAssessmentData";
+
+export function describeModelPerformanceStats(
```
[the remark this answers is not in the document]
Comment (romanlutz) [reply apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeModelPerformanceStats.ts:6]: Ilya created one with this dataset. However, we need to try lots of edge cases that notebooks cannot cover, so this and variations of this dataset will be needed.

File: apps/dashboard/src/error-analysis/__mock_data__/dummyTreeWine.ts (hunk 1 above)
Comment (romanlutz) [reply apps/dashboard/src/error-analysis/__mock_data__/dummyTreeWine.ts:4]: opened #1192 for this.

File: apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeAxisConfigDialog.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (romanlutz) [reply apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeAxisConfigDialog.ts:4]: Because the chart here will change, and so will the tests. I don't think we'll actually change the interpret charts. If you refer to my document, you'll find that there will be a range of different chart options.
